Remove_POS_hindi_text/hindi_POS_tag_removal.py

```diff
@@ -0,0 +1,22 @@
+import string
+import nltk
+
+fp=open(r"C:\Users\ZAVERI SANYA\Desktop\Amazing-Python-Scripts\Remove_POS_hindi_text\\Tagged_Hindi_Corpus.txt",mode="r",encoding="utf-8") #opens the hindi_tagged_corpus.txt file
```

Comment:
Take the inputs of file path of Corpus and only_hindi text files from the user. Also, upload the corpus files to this PR so that I can test the script

Reply:
I have already uploaded the corpus files. Please check `Remove_POS_hindi_text/Tagged_Hindi_Corpus.txt` file @kaustubhgupta

Reply:
Oops, Apologies for that! (Didn't notice). Just update the inputs

Reply:
Okay sir, I will do that!

Reply:
Sir, I have made the recommended changes. Please check! @kaustubhgupta